lib/CL/devices/level0/npu_dbk.h

```diff
@@ -16,3 +15,7 @@ bool instantiateTemplateMATMUL(const void* KernelAttrs,
 bool instantiateTemplateGEMM(const void* KernelAttrs,
                              std::string &ModelXMLInstance,
                              std::string &BuildFlagsInstance);
+
+bool instantiateTemplateCONVERT (const void *KernelAttrs,
```

Comment:
Extra space before parenthesis?

Reply:
Addressed. Renamed the file to `npu_dbk.hh` for the formatter script treats it as C++ source.